src/panels/lovelace/cards/hui-calendar-card.ts

```diff
@@ -187,6 +208,26 @@ export class HuiCalendarCard extends LitElement implements LovelaceCard {
       return;
     }
     this._narrow = card.offsetWidth < 870;
+
+    const header = this.shadowRoot!.querySelector<HTMLDivElement>(".header");
```

Comment:
If the card has a fixed height, can't we just set the height to 100% to fill it?

And does this also work for when it is not used in a section view, so when there isnt a fixed height?

Reply:
Turns out the fix was just this, remove the variable and replace it with a height to fill the rest of the card with an allowance for the header padding